package.json

```diff
@@ -2,18 +2,23 @@
   "name": "final-server",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "type": "module",
+  "main": "v2/index.js",
   "scripts": {
-    "start": "node index.js"
+    "start": "nodemon v2/index.js"
   },
   "dependencies": {
+    "bcryptjs": "^2.4.3",
     "cors": "^2.8.5",
-    "dotenv": "^16.0.3",
+    "crypto": "^1.0.1",
+    "dotenv": "^16.4.1",
     "express": "^4.18.2",
     "jsonwebtoken": "^8.5.1",
-    "mongodb": "^4.11.0",
-    "nodemon": "^2.0.20",
-    "stripe": "^11.1.0"
+    "mongodb": "^4.17.2",
+    "mongoose": "^8.1.1",
+    "nodemailer": "^6.9.9",
+    "nodemon": "^2.0.22",
```

Comment:
Nodemon is the package only we need as dev dependency. You can uninstall it first then install this package as dev dependency.